components/object-property-list/README.md

```diff
@@ -63,8 +79,7 @@ The object property list is designed to wrap in an inline manner if the items ar
 
 The `d2l-object-property-list-item` component is the basic type of item for an object property list, displaying text and an optional leading icon.
 
-
-<!-- docs: demo code -->
+<!-- docs: demo live name:d2l-object-property-list-item -->
```

Comment:
Cool that you can do this now!

Reply:
Yeah - this one actually should (probably?) have worked even before my fixes, I just better understand how they work now than when I first wrote these docs 😄.